components/AudioPlayer/AudioInterface.js

```diff
@@ -0,0 +1,259 @@
+    background: var(--accent-color);
+  }
+`;
+
+const ImagesliderWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 5px 0;
+`;
+
+const MusicTitel = styled.p`
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  color: var(--text-color);
+`;
+
+export default function AudioInterface() {
+  const [hovered, setHovered] = useState(false);
+  const [currentTrackIndex, setCurrentTrackIndex] = useState(0);
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [currentTime, setCurrentTime] = useState(0);
+  const audioRef = useRef(null);
+  const [userInteraction, setUserInteraction] = useState(false);
+  const [volume, setVolume] = useState(0.1);
+
+  useEffect(() => {
+    if (isPlaying) {
+      audioRef.current.play();
+    } else {
+      audioRef.current.pause();
+    }
+  }, [isPlaying]);
+
+  useEffect(() => {
+    audioRef.current.src = playlist[currentTrackIndex].url;
+    audioRef.current.load();
+    audioRef.current.volume = volume;
+    if (isPlaying && userInteraction) {
+      audioRef.current.play().catch((error) => console.error(error));
+    }
+  }, [currentTrackIndex]);
+
+  function handlePlayPause() {
+    if (!userInteraction) setUserInteraction(true);
+    setIsPlaying(!isPlaying);
+  }
+
+  function handlePrevious() {
+    setCurrentTrackIndex((prevIndex) => {
+      if (prevIndex > 0) {
+        return prevIndex - 1;
+      } else {
+        return playlist.length - 1;
+      }
+    });
+  }
+
+  function handleNext() {
+    setCurrentTrackIndex((prevIndex) => {
+      if (prevIndex < playlist.length - 1) {
+        return prevIndex + 1;
+      } else {
+        return 0;
+      }
+    });
+  }
+
+  function handleSliderChange(event) {
+    const newVolume = parseFloat(event.target.value);
+    setVolume(newVolume);
+    audioRef.current.volume = newVolume;
+  }
+
+  const handleTimeUpdate = () => {
+    setCurrentTime(audioRef.current.currentTime);
+  };
+
+  const handleTimeSeek = (event) => {
+    audioRef.current.currentTime = event.target.value;
+    setCurrentTime(audioRef.current.currentTime);
+  };
+
+  return (
+    <>
+      <audio
+        ref={audioRef}
+        onTimeUpdate={handleTimeUpdate}
+        onEnded={handleNext} // Automatically play the next track when current one ends
+      ></audio>
+      <ActivatingInterfaceButton
+        onMouseEnter={() => setHovered(true)}
+        onTouchStart={() => setHovered(true)}
+      >
+        <Image src={musicIcon} alt="MusicIcon" width={40} height={40}></Image>
+      </ActivatingInterfaceButton>
+      {hovered && (
+        <AudioInterfaceWraper onMouseLeave={() => setHovered(false)}>
+          <AudioInterfaceContainer>
+            <ImagesliderWrapper>
+              <Image
+                src={volume === 0 ? volumeOffIcon : volumeUpIcon}
+                alt={`volume ${volume === 0 ? `Off` : `Up`} Icon`}
+                width={20}
+                height={20}
+              ></Image>
```

Comment:
I would self close all the Image components seeing as they don't have children. 

```suggestion
              <Image
                src={volume === 0 ? volumeOffIcon : volumeUpIcon}
                alt={`volume ${volume === 0 ? `Off` : `Up`} Icon`}
                width={20}
                height={20}
              />
```